jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1979,8 +1979,8 @@ properties:
             type: string
             fits_keyword: S_LINEAR
             blend_table: True
-          master_back_sub:
-            title: Master Background subtraction
+          master_background:
```

Comment:
This change now requires a change to https://jira.stsci.edu/browse/JWSTKD-300. The data model name needs to be updated in the "Additional Fields" tab of the form.

Reply:
I updated this in the JWSTKD ticket.